src/Form/AnswersSet.php

```diff
@@ -103,10 +109,19 @@ public static function displayTabContentForItem(
         }
 
         Search::showList(self::class, [
+            'criteria' => [
+                [
+                    'link'       => 'AND',
+                    'field'      => 5,  // Parent form
+                    'searchtype' => 'equals',
+                    'value'      => $item->getID()
+                ]
+            ],
```

Comment:
Fixed an issue where all answers would be shown instead of the answers created by the given form.